src/block/block_open.c

```diff
@@ -272,6 +272,9 @@ __wt_block_open(WT_SESSION_IMPL *session, const char *filename, uint32_t objecti
     if (!forced_salvage)
         WT_ERR(__desc_read(session, allocsize, block));
 
+    /* The block should not be compacted at that time. */
```

Comment:
I didn't clearly understand the comment.

Reply:
I agree that the comment is not the best, it probably does not even deserve a comment. I just want to init the value when the block is opened.

Reply:
No comment would be good.

Reply:
See 7834d1717047a3ff7e6d557e2c699addd0996c02.